openhands/resolver/README.md

```diff
@@ -36,12 +36,13 @@ Follow these steps to use this workflow in your own repository:
    - Optional:
      - `PAT_USERNAME`: GitHub username for the personal access token
      - `PAT_TOKEN`: The personal access token
-     - `LLM_MODEL`: LLM model to use (defaults to "anthropic/claude-3-5-sonnet-20241022")
```

Comment:
This was removed in this PR: https://github.com/All-Hands-AI/OpenHands/pull/5477